Source: https://github.com/mysql/mysql-server/pull/3/commits
Upstream: http://bugs.mysql.com/bug.php?id=75623
Reason: Fix version regex for OpenSSL 1.0.2

From 600fc63d7f5424bff531ae78f400d43682bad1f9 Mon Sep 17 00:00:00 2001
From: Mike McQuaid <mike@mikemcquaid.com>
Date: Sun, 25 Jan 2015 11:37:10 +0000
Subject: [PATCH] ssl.cmake: fix version regex for OpenSSL 1.0.2.

OpenSSL 1.0.2 has whitespace between the # and define so make the regex more
liberal to match it accordingly.
---
 cmake/ssl.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/ssl.cmake b/cmake/ssl.cmake
index 1b2a172..043545c 100644
--- a/cmake/ssl.cmake
+++ b/cmake/ssl.cmake
@@ -166,7 +166,7 @@ MACRO (MYSQL_CHECK_SSL)
     # Encoded as MNNFFPPS: major minor fix patch status
     FILE(STRINGS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h"
       OPENSSL_VERSION_NUMBER
-      REGEX "^#define[\t ]+OPENSSL_VERSION_NUMBER[\t ]+0x[0-9].*"
+      REGEX "^#[\t ]+define[\t ]+OPENSSL_VERSION_NUMBER[\t ]+0x[0-9].*"
     )
     STRING(REGEX REPLACE
       "^.*OPENSSL_VERSION_NUMBER[\t ]+0x([0-9]).*$" "\\1"
